### PR TITLE
feat(platform): aws graviton

### LINKED ISSUE #1040: Test on AWS Graviton
https://aws.amazon.com/ec2/graviton/

> Free Trial: Until Dec 31st 2020, all new and existing AWS customers can try the t4g.micro instances free for up to 750 hours per month. During the free-trial period, customers who run a t4g.micro instance will automatically get 750 free hours per month deducted from their monthly bill through Dec 2020.

https://aws.amazon.com/ec2/faqs/#t4g-instances

We expect the test to fail before additional work has been done in the Client team.

### PR DESCRIPTION
**Pull request by divyenduz**
Fixes https://github.com/prisma/e2e-tests/issues/1040

The binary fails as expected

```
Error: Get config /home/ec2-user/aws-graviton/node_modules/@prisma/cli/query-engine-rhel-openssl-1.0.x: /home/ec2-user/aws-graviton/node_modules/@prisma/cli/query-engine-rhel-openssl-1.0.x: cannot execute binary file
```

```
[ec2-user@ip-172-31-20-254 aws-graviton]$ cat /etc/os-release 
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
```

### CODE REVIEW

File: platforms/aws-graviton/prepare.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+
+cp ./code/package.json ./package.json
+
+if [ "$OSTYPE" = "linux" ]; then
```
Comment (janpio) [inline platforms/aws-graviton/prepare.sh]: Why is this needed?

**Comment by janpio** [comment]
Waiting for: https://github.com/prisma/prisma/issues/4573

File: platforms/aws-graviton/prepare.sh (hunk 1 above)
Comment (divyenduz) [reply platforms/aws-graviton/prepare.sh]: Probably not required anymore, there was some command that I was using that only worked on `linux`. I will validate that and remove it.
